### PR TITLE
Update search.md

### PR DESCRIPTION
**Pull request by shubham75015**
### What does this PR do?

added support for measures
### Motivation

new feature
### Preview link

https://docs-staging.datadoghq.com/Shubham/MeasureFacets/tracing/app_analytics/search/

https://docs-staging.datadoghq.com/Shubham/MeasureFacets/tracing/app_analytics/search/

### CODE REVIEW

File: content/en/tracing/app_analytics/search.md [hunk 1]
```diff
@@ -172,6 +172,23 @@ Facets allow you to pivot or filter your datasets based on a given attribute. Ex
 
 {{< img src="tracing/app_analytics/search/facets_demo.png" alt="Facets demo"  style="width:80%;">}}
 
+### Quantitative facets: Measures
```
Comment (ruthnaebeck) [inline content/en/tracing/app_analytics/search.md]:
```suggestion
### Quantitative facets: measures
```

File: content/en/tracing/app_analytics/search.md [hunk 2]
```diff
@@ -172,6 +172,23 @@ Facets allow you to pivot or filter your datasets based on a given attribute. Ex
...
+
+**Use measures when you need:**
```
Comment (ruthnaebeck) [inline content/en/tracing/app_analytics/search.md]:
```suggestion
**Use measures when you need to:** 
```

File: content/en/tracing/app_analytics/search.md [hunk 3]
```diff
@@ -172,6 +172,23 @@ Facets allow you to pivot or filter your datasets based on a given attribute. Ex
 
 {{< img src="tracing/app_analytics/search/facets_demo.png" alt="Facets demo"  style="width:80%;">}}
 
+### Quantitative facets: Measures
+
+**Use measures when you need:** 
+* To aggregate values from multiple Traces. For instance, create a measure on the number of rows in Cassandra and view the P95 or top-most referrers per sum of file size requested.
```
Comment (ruthnaebeck) [inline content/en/tracing/app_analytics/search.md]:
```suggestion
* Aggregate values from multiple traces. For example, create a measure on the number of rows in Cassandra and view the P95 or top-most referrers per sum of file size requested.
```

File: content/en/tracing/app_analytics/search.md [hunk 4]
```diff
@@ -172,6 +172,23 @@ Facets allow you to pivot or filter your datasets based on a given attribute. Ex
...
+* To numerically compute the highest latency services for shopping cart values over $1000
```
Comment (ruthnaebeck) [inline content/en/tracing/app_analytics/search.md]:
```suggestion
* Numerically compute the highest latency services for shopping cart values over $1000.
```

File: content/en/tracing/app_analytics/search.md [hunk 5]
```diff
@@ -172,6 +172,23 @@ Facets allow you to pivot or filter your datasets based on a given attribute. Ex
...
+*To filter continuous values. For instance, size in bytes of each payload chunk of a video stream
```
Comment (ruthnaebeck) [inline content/en/tracing/app_analytics/search.md]:
```suggestion
* Filter continuous values. For example, the size in bytes of each payload chunk of a video stream.	
```

File: content/en/tracing/app_analytics/search.md [hunk 6]
```diff
@@ -172,6 +172,23 @@ Facets allow you to pivot or filter your datasets based on a given attribute. Ex
...
+* To numerically compute the highest latency services for shopping cart values over $1000
+*To filter continuous values. For instance, size in bytes of each payload chunk of a video stream	
+
+**Types**
+
+Measures come with either a (long) integer or double value, for equivalent capabilities.
+
+**Units**
+
+Measures support units (time in seconds or size in bytes) for easier handling of orders of magnitude at query time and display time. Unit is a property of the measure itself, not of the field. For example, consider a duration measure in nanoseconds: you have a span tag from `service:A` where `duration:1000` stands for `1000 milliseconds`, and other span tags from `service:B` where `duration:500` stands for `500 microseconds`:
```
Comment (ruthnaebeck) [inline content/en/tracing/app_analytics/search.md]:
```suggestion
Measures support units (time in seconds or size in bytes) for handling of orders of magnitude at query time and display time. Unit is a property of the measure itself, not of the field. For example, consider a duration measure in nanoseconds: you have a span tag from `service:A` where `duration:1000` stands for `1000 milliseconds`, and another span tags from `service:B` where `duration:500` stands for `500 microseconds`:
```

File: content/en/tracing/app_analytics/search.md [hunk 7]
```diff
@@ -172,6 +172,23 @@ Facets allow you to pivot or filter your datasets based on a given attribute. Ex
...
+Scale duration into nanoseconds for all span tags flowing in with the arithmetic processor. Use a `*1000000` multiplier on span tags from `service:A`, and a `*1000` multiplier on span tags from `service:B`.
+Use `duration:>20ms` (see search syntax for reference) to consistently query span tags from both services at once, and see an aggregated result of max 1 min.
```
Comment (ruthnaebeck) [inline content/en/tracing/app_analytics/search.md]:
```suggestion
Use `duration:>20ms` (see search syntax for reference) to consistently query span tags from both services at once, and see an aggregated result of max one minute.
```

**Comment by shubham75015** [comment]
thank you - we can merge in prod.

**Pull request merged**